docs/python-api.md

```diff
@@ -69,7 +69,18 @@ response = model.prompt(
     ]
 )
 ```
-Use `llm.Attachment(content=b"binary image content here")` to pass binary content directly.
+Use `llm.Attachment(content=b"binary image content here")` to pass binary content directly. You may optionqlly specify a content type - if tou do not it will be detected automatically:
```

Comment:
```suggestion
Use `llm.Attachment(content=b"binary image content here")` to pass binary content directly. You may optionally specify a content type - if you do not it will be detected automatically:
```